libs/backend/src/scan/cast_vote_records/export.ts

```diff
@@ -404,14 +429,20 @@ export async function exportCastVoteRecordsToUsbDrive(
     }
     const { castVoteRecordId, castVoteRecordHash } =
       exportCastVoteRecordFilesResult.ok();
+    castVoteRecordHashes[castVoteRecordId] = castVoteRecordHash;
+  }
+
+  for (const [castVoteRecordId, castVoteRecordHash] of Object.entries(
+    castVoteRecordHashes
+  )) {
```

Comment:
An interesting bug here. I had to move the updating of cast vote record hashes out of the result-sheet iterating loop to avoid SQLite "DB client is busy" errors. There was no issue before when I'd only used this function to export one result sheet at a time for continuous export, as the result-sheet iterating query was always finished by the time we ran hash-updating queries.

I have tests later in this PR that will make sure that we don't accidentally regress here.

Reply:
🐛